portfolio.html

```diff
@@ -50,6 +50,22 @@ <h3>Who am I?</h3>
 	</section>
 	<h2 id="Contact">Contact</h2>
 	<footer class="contactSection">
+		<div class="contactSectionContainer">
+			<h3>Contact details</h3>
+			<p>The best way to contact me is through email, though I'm also on Linkedin and Github.</p>
+			<div>
+				<p>My Email is: <strong>garrettethan63@gmail.com</strong></p>
+				<img src="emailIcon.png" alt ="" class="contactHide">
```

Comment:
Is this image icon supposed to disappear when I reduce my screen width? Also, how about having the images spread across horizontally when the screen is smaller to leave less empty space?

Reply:
Email icon is supposed to disappear as it is not required so to save space on smaller screens it disappears